storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst

```diff
@@ -35,31 +35,45 @@ Setting up a slave for replication with |Percona XtraBackup| is really a very st
 * ``Percona XtraBackup``
   The backup tool we will use. It should be installed in both computers for convenience.
 
+.. note:: It is not recommended to mix MySQL variants
+   (Percona Server, MySQL, MariaDB) in your replication setup.
+   This may produce incorrect :file:`xtrabackup_slave_info` file
+   when adding a new slave.
+
 STEP 1: Make a backup on ``TheMaster`` and prepare it
 =====================================================
 
 At ``TheMaster``, issue the following to a shell:
 
 .. code-block:: console
 
-   TheMaster$ innobackupex --user=yourDBuser --password=MaGiCdB1 /path/to/backupdir 
+   TheMaster$ xtrabackup --backup --user=yourDBuser --password=MaGiCdB1 --target-dir=/path/to/backupdir 
 
 After this is finished you should get:
 
 .. code-block:: console
 
-   innobackupex: completed OK! 
+   xtrabackup: completed OK! 
 
-This will make a copy of your |MySQL| data dir to the /path/to/backupdir/$TIMESTAMP. You have told |Percona XtraBackup| (through the |innobackupex| script) to connect to the database server using your database user and password, and do a hot backup of all your data in it (all |MyISAM|, |InnoDB| tables and indexes in them).
+This will make a copy of your |MySQL| data dir
+to the :file:`/path/to/backupdir` directory.
+You have told |Percona XtraBackup| to connect to the database server
+using your database user and password,
+and do a hot backup of all your data in it
+(all |MyISAM|, |InnoDB| tables and indexes in them).
 
 In order for snapshot to be consistent you need to prepare the data:
 
 .. code-block:: console
 
-   TheMaster$ innobackupex --user=yourDBuser --password=MaGiCdB1 \
-              --apply-log /path/to/backupdir/$TIMESTAMP/
+   TheMaster$ xtrabackup --user=yourDBuser --password=MaGiCdB1 \
+                --prepare --target-dir=/path/to/backupdir
 
-You need to select path where your snapshot has been taken, for example /home/backups/2012-01-16_11-14-43. If everything is ok you should get the same OK message. Now the transaction logs are applied to the data files, and new ones are created: your data files are ready to be used by the MySQL server.
+You need to select path where your snapshot has been taken.
+If everything is ok you should get the same OK message.
+Now the transaction logs are applied to the data files,
+and new ones are created:
+your data files are ready to be used by the MySQL server.
 
 |Percona XtraBackup| knows where your data is by reading your :term:`my.cnf`. If you have your configuration file in a non-standard place, you should use the flag :option:`--defaults-file` ``=/location/of/my.cnf``.
```

Comment:
Since MySQL 5.6 `.mylogin.cnf` is recommended place to store user credentials instead of `my.cnf`. PXB does support it. https://dev.mysql.com/doc/refman/5.6/en/mysql-config-editor.html